### PR TITLE
Release 2021.09.2

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adding release notes for 2021.09.2
- **Is this PR related to a Monolith release?** Yes. 2021.09.2

### Resolves:

* [OKTA-427510](https://oktainc.atlassian.net/browse/OKTA-427510)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+
+### Weekly Release 2021.09.2
+
+| Change                                                                         | Expected in Preview Orgs |
+|--------------------------------------------------------------------------------|--------------------------|
+| [Audience parameter deprecated for the Authentication API](#audience-parameter-deprecated-for-the-authentication-api)       | September 15, 2021             |
+
+#### Audience parameter deprecated for the Authentication API
+
+The `audience` parameter of the `/authn` [endpoint](/docs/reference/api/authn/#primary-authentication) has been deprecated. <!--OKTA-385098-->
```
[the remark this answers is not in the document]
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:14]: I think you have to use the full path if it's not an OAuth endpoint? Confirm with the style guide.

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 2]
```diff
@@ -1466,6 +1466,8 @@ Authenticates a user for signing in to the specified application
 * Only WS-Federation, SAML based apps are supported.
 * Pass the application instance ID of the app as ["audience"](#request-parameters-for-primary-authentication) along with the user credentials.
 
+> **Note:** `audience` is a <ApiLifecycle access="deprecated" /> property.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:1469]: I thought `audience` was a parameter? (or is it also a property?)

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 3]
```diff
@@ -1486,6 +1488,8 @@ curl -v -X POST \
 }' "https://${yourOktaDomain}/api/v1/authn"
 ```
 
+> **Note:** `audience` is a <ApiLifecycle access="deprecated" /> property.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:1491]: same comment: is `audience` a parameter or a property?
Also, the developer should provide a new snippet without `audience` if it's deprecated? Is there a reason why we want to keep this here?

**Review by vanngo-okta — APPROVED** [review]
Just some small nits if you have time to ask the developer. The release notes look good! 🚢

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:14]: @vanngo-okta I confirmed with the style guide when I wrote it:
> When you mention an endpoint in any Developer Doc content (including both bugs, features, and enhancements in the API Release Notes), always include the full path that follows /api/v1/ for all endpoints except OAuth endpoints. For OAuth endpoints, always include the full path that follows /v1/.

I take that to mean include everything after `/api/v1/`. This appears to be consistent with what the rest of the release notes say :)

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md (hunk 3 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authn/index.md:1491]: I'll follow up with the developer but, based on what I see in the related Jira ticket, they don't want to remove the functionality yet. So, technically, they're in the process of deprecating it. [OKTA-385098](https://oktainc.atlassian.net/browse/OKTA-385098)

**Pull request merged**